.bazelignore

```diff
@@ -0,0 +1 @@
+examples
```

Comment:
note, this is needed as otherwise Bazel in the outer workspace isn't smart enough to exclude the nested workspace from a `//...` expansion.